python/pyspark/pandas/namespace.py

```diff
@@ -238,7 +238,8 @@ def read_csv(
     path : str
         The path string storing the CSV file to be read.
     sep : str, default ‘,’
-        Delimiter to use. Must be a single character.
+        Delimiter to use. A character string(len(sep) >= 1) which is used to separate
```

Comment:
Space after 'string' (or just omit the parenthetical)
You're sure a multi-character string works here?

Reply:
@bzhaoopenstack I like this one.. 
 
Delimiter to use. Can also use a custom delimiter as separator.

Reply:
> OK, fix the spacing at least, or just say "nonempty string"

Yeah, I test on my local env and compare the output with pandas. ;-) . It works and depends on the spark native DataFrame ability.

And I agree with you about the simple description looks better, thanks.

Reply:
> @bzhaoopenstack I like this one..
> 
> Delimiter to use. Can also use a custom delimiter as separator.

Yeah, thanks